src/Serilog.Sinks.AzureTableStorage/Sinks/AzureTableStorageWithProperties/AzureTableStorageEntityFactory.cs

```diff
@@ -102,5 +104,17 @@ private static bool IsValidColumnName(string propertyName)
 
             return isValid;
         }
+
+        /// <summary>
+        /// Determines if the given property name exists in the specific columns. 
+        /// Note: If specific columns is not defined then the property name is considered valid. 
+        /// </summary>
+        /// <param name="propertyName">Name of the property to check</param>
+        /// <param name="specificProperties">List of defined properties only to be added as columns</param>
+        /// <returns>true if the no specificColumns are specified or it is included in the specificColumns property</returns>
+        private static bool IsValidSpecificColumn(string propertyName, string[] specificProperties)
```

Comment:
This is checking inclusion rather than validity - would a name like `IsPropertyWrittenToColumn()` (okay that's a bit long :-)) be a better description?